k8s/charts/openebs/values.yaml

```diff
@@ -46,7 +46,7 @@ provisioner:
 
 localprovisioner:
   image: "quay.io/openebs/provisioner-localpv"
```

Comment:
Add a new variable called - helperImage: "quay.io/openebs/openebs-tools:3.8"

Update the readme -- with the above value in the localprovisioner section. "The image used to manage the hostpath used by Local PV".

Reply:
done